src/starknet-wrappers.ts

```diff
@@ -433,6 +433,8 @@ export class DockerWrapper extends StarknetWrapper {
         options.feederGatewayUrl = adaptUrl(options.feederGatewayUrl);
         const preparedOptions = this.prepareTxQueryOptions("tx_status", options);
 
+        options.gatewayUrl = adaptUrl(options.gatewayUrl);
```

Comment:
This URL adaptation was fixed with an earlier PR, something might not have been done properly with rebasing to the master branch because the same functionality is done in line 433.

Reply:
https://github.com/Shard-Labs/starknet-hardhat-plugin/pull/106/commits/a8db418f44d0b4677de40c47b45c4e673a043525